lib/graphql/batch/executor.rb

```diff
@@ -20,7 +20,8 @@ def start_batch(executor_class)
       def end_batch
         executor = current
         unless executor
-          raise NoExecutorError, 'Cannot end a batch without an Executor.'
+          raise GraphQL::Batch::NoExecutorError,
```

Comment:
With NoExecutorError defined on GraphQL::Batch there is no need to reference it directly, just using `NoExecutorError` as was done before should be sufficient.